package-lock.json

```diff
@@ -98,18 +98,16 @@
     },
     "node_modules/@aashutoshrathi/word-wrap": {
       "version": "1.2.6",
-      "resolved": "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz",
-      "integrity": "sha512-1Yjs2SvM8TflER/OD3cOjhWWOZb58A2t7wpE2S9XfBYTiIl+XFhQG2bjy4Pu1I+EAlCNUzRDYDdFwFYUKvXcIA==",
       "dev": true,
+      "license": "MIT",
```

Comment:
What caused all of these `package-lock` changes?

Reply:
I installed package `"notifications-node-client": "^8.0.0",` this modified `package-lock.json`.

After this I was merging back `main` branch changes, for `package-lock.json` conflicts I was just deleting the file and regenerating it.